app/controllers/articles_controller.rb

```diff
@@ -1,15 +1,21 @@
 class ArticlesController < ApplicationController
+  before_action :set_users, only: [:index]
+
   def index
-    company = Company.find(params[:company_id])
+    all_articles = Article.article_list(@users.ids).published
```

Comment:
面談の時に上手く言葉にできずにすみませんでした💦

方針としては、
1. company_id からその社員の一覧（ID）を取得
2. 1.で取得した user_id の公開済み記事を取ってくる
という感じで実装しています。

Reply:
https://github.com/ispec-inc/Joy-Recruit/pull/5/files#r402935145
上記ご指摘頂いた点について考えたのですが、
リレーションの `.articles` を使う場合、以下のコードしか思いつかなかったのですが、
これだとN+1問題の解決にはなってないかと思いまして...🤔

```suggestion
    all_articles = []
    @users.each do |user|
      all_articles << user.articles.published
    end
```